src/LoggerAbstractServiceFactory.php

```diff
@@ -26,10 +26,18 @@ class LoggerAbstractServiceFactory implements AbstractFactoryInterface
      *
      * @var string
      */
-    protected $configKey = 'log';
+    protected $configKey;
```

Comment:
Potentially problematic, if child classes override the constructor, but I don't think we should stress it: accessing state before the parent constructor is called would be a mistake anyway :)